docs/google-analytics.md

```diff
@@ -5,11 +5,9 @@ This project includes a Google Analytics set up with custom events for CSV downl
 Two custom events are included to help track how users are interacting with the product. 
 
 ## CSV Download
-![alt text](download-csv.png "Example image showing download CSV events in Google Analytics")
 
 There are two events for CSV downloads: full CSV & specific domain CSV. This are categorized under "Download" actions.
```

Comment:
Are they called "event actions"? Could be helpful to include how to get there (Behavior -> Events -> Top Events -> "Download").